lighthouse-core/computed/metrics/lantern-metric.js

```diff
@@ -15,6 +15,15 @@ const LoadSimulator = require('../load-simulator.js');
 /** @typedef {import('../../lib/dependency-graph/network-node')} NetworkNode */
 /** @typedef {import('../../lib/dependency-graph/simulator/simulator')} Simulator */
 
+/**
+ * @typedef Extras
+ * @property {boolean} optimistic
+ * @property {LH.Artifacts.LanternMetric} fcpResult
+ * @property {LH.Artifacts.LanternMetric} fmpResult
+ * @property {LH.Artifacts.LanternMetric} interactiveResult
+ * @property {{speedIndex: number}} speedline
```

Comment:
should these all (except `optimistic`) be optional, though?

Reply:
done